docs/1.guide/0.index.md

```diff
@@ -0,0 +1,58 @@
+---
+icon: pixel:play
+---
+
+# Getting Started
+
+> Get to know srvx and familiarize yourself with basic usage.
+
+Simply put, **srvx** is a compatibility layer for Node APIs. What this means is that it acts as a unified standard API to create HTTP servers based on the standard web platform primitives ([Fetch][Fetch], [Request][Request], and [Response][Response]) which integrates seamlessly with [Deno][Deno], [Bun][Bun], [Node.js][Node], and more.
```

Comment:
Srvx is not compatibility layer for Node.js but unifying layer for web standards.